app/models/user.rb

```diff
@@ -120,6 +120,7 @@ def sync_profile
   # we keep page views as they belong to the article, not to the user who viewed it
   has_many :page_views, dependent: :nullify
 
+  has_many :podcasts, through: :podcast_ownerships
```

Comment:
as https://github.com/MLH-Fellowship/dev.to/issues/82 is coming down the pipeline, the association here should be named `:owned_podcasts`, so that a user can be the owner of one or more podcasts and also appear in one or more podcasts

Reply:
@rhymes can we check this once with @benhalpern before changing it? Because in my previous meeting with Ben and Chris, we had decided on `podcast_ownerships`

Reply:
@Manasa2850 what I meant is changing `user.podcasts` to `user.owned_podcasts` or `user.podcasts_owned`

that's because https://github.com/forem/forem/pull/11354 will add the concept of "podcasts that I have been a guest on" which is different from "podcasts that I owned and manage". They can't be both called `podcasts`. The name `through :podcast_ownerships` is correct :)